includes/Block/Web_Stories_Block.php

```diff
@@ -87,10 +80,25 @@ public function register() {
 			$this->get_script_settings()
 		);
 
+		$base_path = $this->assets->get_base_path( 'assets/js/block.json' );
```

Comment:
Is there a reason why this call isn't just within `register_block_type()`? Seems a bit superfluous to have this `$base_path` param.